scripts/pre_commit_linter.py

```diff
@@ -580,6 +580,98 @@ def _check_bad_pattern_in_file(filename, content, pattern):
     return False
 
 
+def _check_bad_pattern_in_comments(filename, content):
```

Comment:
After a second think, it would be better to take the approach that was done for `_check_import_order`, so trace how that was implemented beginning from the `main` function and try to adapt it for your check here.

Reply:
The main reason is that the implementation here isn't very efficient, so that implementation mentioned above is a good example.